src/colorizer/CanvasOverlay.ts

```diff
@@ -241,34 +254,58 @@ export default class CanvasOverlay implements IRenderCanvas {
   }
 
   public setZoom(zoom: number): void {
-    this.zoomMultiplier = zoom;
+    // TODO: Replace all these checks for specific instances with canvas type (2D/3D)
```

Comment:
does this comment mean to use `this.innerCanvasType` instead of `instanceof`?

Reply:
Yes! I need to think through the typing a little more though so I'm leaving this for a future change.